File: 其他Demo/LuaSpaceShoot-master/Assets/Lua/SpaceShoot/BoundaryCube.lua
BoundaryCube = {}

local this = BoundaryCube;
--游戏中的边界碰撞检测逻辑（子弹或者Cube离开游戏界面是销毁功能
function this.OnTriggerExit(other)
	print("OnTriggerExit(other");
	this.gameObject.Destroy(other.gameObject);
end

File: 其他Demo/LuaSpaceShoot-master/Assets/Scripts/SpaceShoot/BoundaryCube.cs
using UnityEngine;
using System.Collections;

public class BoundaryCube : LuaBehaviour {

    void Awake()
    {
        base.Init("SpaceShoot/BoundaryCube");
        base.Awake();
    }

    void OnTriggerExit(Collider other)
    {
        CallMethod("OnTriggerExit", other);
        //Destroy(other.gameObject);
    }

     
}


